keyboards/keebio/iris/keymaps/mojitas/keymap.c

```diff
@@ -0,0 +1,284 @@
+#include QMK_KEYBOARD_H
+
+//Heavily modified keymap. Some features:
+//Multiple layouts, I use dvorak as main.
+//Nordic(swedish) signs
+//Symbols, numpad, arrows/navigation reachable under the alpas via the layers
+//Mirrored ctl, alt and shift to be able to use both hands when doing commands
+//Gaming layer, qwerty with space on left half.
+
+#define _QWERTY 2
+#define _DVORAK 0
+#define _COLEMAK 1
+#define _WORKMAN 3
+#define _GAMING 4
+#define _NUMPAD 5
+#define _LOWER 6
+#define _RAISE 7
+
+#define _ADJUST 16
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  DVORAK,
+  COLEMAK,
+  WORKMAN,
+  GAMING,
+  NUMPAD,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+
+#define KC_ KC_TRNS
+
+#define KC_Sw2 RALT(KC_2) // Nordic @
+#define KC_Sw3 RALT(KC_3) // Nordic something
+#define KC_Sw4 RALT(KC_4) // Nordic something
+#define KC_Sw5 RALT(KC_5) // Nordic something
+#define KC_Sw6 RALT(KC_6) // ...
+#define KC_Sw7 RALT(KC_7)
+#define KC_Sw8 RALT(KC_8)
+#define KC_Sw9 RALT(KC_9)
+#define KC_Sw0 RALT(KC_0)
+#define KC_Tild RALT(KC_RBRC)
+#define KC_Bsls RALT(KC_MINS)
+#define KC_Bar RALT(KC_NUBS)
+#define KC_Less S(KC_NUBS)
+#define KC_CATDEL LCTL(LALT(KC_DEL)) // Ctrl alt del
+#define KC_TSKMGR LCTL(S(KC_ESC)) // Ctrl shift esc
+#define KC_NUMP TG(_NUMPAD)  // Toggle layer NUMPAD for use in LAYOUT_kc
+#define KC_Close RALT(KC_F4) // Alt F4
+#define KC_Great S(KC_NUBS)
+#define KC_MEH1 MEH(KC_1)
+#define KC_MEH2 MEH(KC_2)
+#define KC_MEH3 MEH(KC_3)
+#define KC_MEH4 MEH(KC_4)
+#define KC_MEH5 MEH(KC_5)
+
+#define KC_LOWR LOWER
+#define KC_RASE RAISE
+#define KC_RST RESET
+#define KC_BL_S BL_STEP
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [_QWERTY] = LAYOUT_kc(
```

Comment:
Layout macros that automatically prepend `KC_` are discouraged, as they do not play well with the Configurator, and make some of the more advanced keycodes such as `MO()` and `LT()` harder to use, since you need to manually define aliases beginning with `KC_` (as you've done here).